doc/manual/de/config/url-params.rst

```diff
@@ -109,7 +109,7 @@ um dies fest einzustellen.
 .. code::
 
     Default: cgi-bin (backend=cgi-bin)
-    Options: cgi-bin (backend=cgi-bin), oh (backend=oh), oh2 (backend=oh2)
+    Options: cgi-bin (backend=cgi-bin), oh (backend=oh), oh2 (backend=oh2), mqtt
```

Comment:
we should go with "openhab" here, all the other variants are currently kept for backwards compability and should be removes with the next library upgrade